test/verify/check-connection

```diff
@@ -109,7 +109,7 @@ class TestConnection(MachineCase):
             self.assertTrue(cookie["httpOnly"])
         self.assertFalse(cookie["secure"])
 
-        if not m.atomic_image:  # cannot write to /usr on Atomics, and cockpit-session is in a container
```

Comment:
You changed all of these from `if [not] m.atomic_image` to `if [not] m.ostree_image` but it does not make really sense.

`atomic_image` was `m.image == "rhel-atomic"`. Now you converted it into `ostree_image` which is `m.image == "fedora-coreos"`.

I think all of these conditions can be dropped.

Reply:
I tried to keep the underlying reason for those. Atomic was OSTree as well, and most of these special cases were about "non-writable /usr" or "local login uses cockpit-ssh" and such. As such, I found these changes less intrusive (while still conceptually correct) than dropping them all.

But of course we won't actually start testing the rhel-7 branch on CoreOS, so we may just as well remove all the special cases. That's going to be a bigger diff noise.

Reply:
OK, dropped all Atomic special cases now.